### DISCUSSION TITLE
重新Deploy后提示找不到think-model-mysql 模块

### DISCUSSION THREAD

**Discussion by elkan1788** [walinejs/waline#1335] [Q&A] [answered]
刚在Vercel上面更新了一些变更信息，结果在重新Deploy时出现如何的错误：
![image](https://user-images.githubusercontent.com/1217096/187364694-6e02b606-1ca6-40b8-8f90-f1c5ce4a8da9.png)

这个算是Bug还是什么，是之前是直接使用手册上面的模板在vercel上面部署的，没有调整过配置文件。

```json
{
  "name": "template",
  "version": "0.0.1",
  "private": true,
  "dependencies": {
    "@waline/vercel": "latest"
  }
}
```

**Comment by elkan1788** [comment]
看起来应该是个Bug，重新Deploy一个新的项目也是报错的呀

**Reply by hxer520** [reply]
我也遇到同样问题

**Reply by elkan1788** [reply]
估计是vercel的问题，尝试过回退之前的Waline版本，仍是不能生效，问题还是一样的。

**Comment by lizheming** [comment, marked as answer]
https://github.com/walinejs/waline/issues/1334
